wg-manifests/charter.md

```diff
@@ -1,70 +1,32 @@
-# WG Manifests Charter
+# WG Platform/Manifests Charter
 
-This charter adheres to the conventions, roles and organization management
-outlined in [wg-governance].
+This platform/manifests charter describes the working mode / reality / status quo of the last 5 years as of March 2025.
+It tries to be as lean as possible and balance community and commercial interests.
```

Comment:
This is a very strange statement, whose commercial interest are we talking about? 

Is it the people who form the working group, or some overarching group, the maintainers of the components, or something else?

Reply:
Well I can remove that sentence, but just in general commercial distributions, commercial support, even employees that work with Kubeflow and therefore get a salary related to kubeflow. It is not limited to Kubeflow organization members.

Reply:
Well let me remove the statement